modules/ROOT/pages/installation/linux/debian.adoc

```diff
@@ -16,7 +16,6 @@ To install Neo4j on Debian you need to make sure of the following:
 
 Neo4j {neo4j-version} requires the Java 11 runtime.
 Java 11 is not included in Ubuntu 16.04 LTS or Debian 9 (stretch) and will have to be set up manually prior to installing or upgrading to Neo4j {neo4j-version}, as described below.
```

Comment:
We may need to remove either this sentence, or the reference to Debian 9 (stretch) here.

@jennyowen perhaps you could advise? 🙇‍♂️

Reply:
I was actually thinking about debian support earlier today!
We need to remove all reference to Debian 9 and update the [software requirements](https://neo4j.com/docs/operations-manual/4.4/installation/requirements/#deployment-requirements-software) to say debian 10+ instead of 9+.